### PR TITLE
CC: Update support location directions

### PR DESCRIPTION
**Pull request by tmjd**
Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: calico-cloud/get-started/connect/operator-checklist.mdx [hunk 1]
```diff
@@ -679,6 +679,6 @@ To install an appropriate Prometheus operator, contact Support below.
 
 ## Next step
 
-If you have verified all of the above steps and still have issues, click the chat icon in Manager UI at the bottom of the left nav to get help from Support:
+If you have verified all of the above steps and still have issues, click the question icon in the UI under the user icon in the upper right to get help from Support:
 
 ![contact-support](/img/calico-cloud/contact-support.png)
```
Comment (ctauchen) [inline calico-cloud/get-started/connect/operator-checklist.mdx:684]: Let's just remove this section.

File: calico-cloud_versioned_docs/version-19-1/get-started/connect/operator-checklist.mdx [hunk 2]
```diff
@@ -679,6 +679,6 @@ To install an appropriate Prometheus operator, contact Support below.
 
 ## Next step
 
-If you have verified all of the above steps and still have issues, click the chat icon in Manager UI at the bottom of the left nav to get help from Support:
+If you have verified all of the above steps and still have issues, click the question icon in the UI under the user icon in the upper right to get help from Support:
 
 ![contact-support](/img/calico-cloud/contact-support.png)
```
Comment (ctauchen) [inline calico-cloud_versioned_docs/version-19-1/get-started/connect/operator-checklist.mdx:684]: rm

**Review by ctauchen — COMMENTED** [review]
Best just to remove this section entirely.

**Comment by tmjd** [comment]
After chatting with Chris a bit I'm going to close this PR.

**Pull request closed without merging**